### PR TITLE
MKM: Intrude on the Mind + 5 cards

### PR DESCRIPTION
**Pull request by MikeS-NZ**
MKM: Add card scripts for six new MKM cards and one token script:

- Buried in the Garden
- Intrude on the Mind
- Kraul Whipcracker
- Long Goodbye
- No More Lies
- Warleader's Call
- Colorless 0/0 Thopter artifact creature token with flying (Intrude on the Mind's token)

Update 1/18: Add script for new MKM card Doppelgang (stowaway commit after updating another script, my bad)

- Doppelgang

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/kraul_whipcracker.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Kraul Whipcracker
+ManaCost:B G
+Types:Creature Insect Assassin
+PT:3/2
+K:Reach
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDestroy | TriggerDescription$ When CARDNAME enters the battlefield, destroy target token an opponent controls.
+SVar:TrigDestroy:DB$ Destroy | ValidTgts$ Permanent.token+OppCtrl | TgtPrompt$ Select target token an opponent controls
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/kraul_whipcracker.txt:7]: wrong params

File: forge-gui/res/cardsfolder/upcoming/long_goodbye.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Name:Long Goodbye
+ManaCost:1 B
+Types:Instant
+K:This spell can't be countered.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/long_goodbye.txt:4]: this KW no longer exists

Comment (MikeS-NZ) [reply forge-gui/res/cardsfolder/upcoming/long_goodbye.txt]: Fixed. Used a reference dated prior to this update. Thanks!

File: forge-gui/res/cardsfolder/upcoming/kraul_whipcracker.txt (hunk 1 above)
Comment (MikeS-NZ) [reply forge-gui/res/cardsfolder/upcoming/kraul_whipcracker.txt:7]: Fixed!

**Review by tool4ever — APPROVED** [review]

**Pull request merged**